src/twisted/python/_sendmsg.c

```diff
@@ -278,6 +278,14 @@ static PyObject *sendmsg_sendmsg(PyObject *self, PyObject *args, PyObject *keywd
                 PyErr_NoMemory();
                 goto finished;
             }
+            /* From Python 3.5.2 socketmodule.c:3891:
+               Need to zero out the buffer as a workaround for glibc's
+               CMSG_NXTHDR() implementation.  After getting the pointer to
+               the next header, it checks its (uninitialized) cmsg_len
+               member to see if the "message" fits in the buffer, and
+               returns NULL if it doesn't.  Zero-filling the buffer
+               ensures that this doesn't happen. */
+            memset(message_header.msg_control, 0, all_data_len);
```

Comment:
From what I can tell from the Python docs, those are not available on Python 2, for which _sendmsg.c is need. Am I mistaken?